WindsorBody/Allrun.pre

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -l
+### Script for preparing the WindsorBody-Case with the c1g1-mesh from 
+### AutoCFD3-workshop
+### Upstream CFD, 2022
+
+# EXIT ON ERROR
+set -e
+
+
+# blockmesh and feature extraction
+prepareMesh=false                       # to setup a new case folder
+devFlowField=true                       # to setup a new case folder with a developed DDES-field
+
+### Set nice level if required
+#NI=" "
+NI="nice -n 19"
+
+### directories
+logDir="logFiles"
+if $devFlowField; then
+    srcDir="WindsorBody_OF-v2206_incomp_AutoCFD3g1_devFlow.orig"
+else
+    srcDir="WindsorBody_OF-v2206_incomp_AutoCFD3g1.orig"
+fi
+runDir="WindsorBody_OF-v2206_incomp_AutoCFD3g1_DDES"
+
+### Load modules as required
+module load gnu/11
```

Comment:
The module structure is different on different machines. I would say we keep a separate folder for things that are machine specific.